stdlib.sh

```diff
@@ -389,6 +453,106 @@ function __STDLIB_oneshot_syntax_check() { # {{{
 
 ###############################################################################
 #
+# stdlib.sh - Initialise coloured output
+#
+###############################################################################
+
+function __STDLIB_oneshot_colours_init() { # {{{
+
+	if(( std_COLOUR_ON )); then
+		local foregroundInfo="white"
+		local backgroundInfo=""
+		local modeInfo=""
```

Comment:
Ooh, I don't like camel-case in shell scripts :(

Also, `foregroundInfo` or `infoforeground` (or `infofg`)?